src/components/SongLibrary/SongLibrary.js

```diff
@@ -21,7 +21,7 @@ class SongLibrary extends Component {
     || song.artist.toUpperCase().includes(modifiedSearchQuery)
     )
 
-    if(matchingSongs != "") {
+    if(matchingSongs !== "") {
```

Comment:
I think this will need to get switched back. The double equals was throwing off the intended functionality when I was testing it on a previous branch.

Reply:
oh ok, React was giving me errors for the single equals, it must prefer the strictly equal which makes sense for built in assurances for developers. I will switch it back, maybe there is a config we can set to allow it